.github/workflows/test.yaml

```diff
@@ -24,22 +28,42 @@ defaults:
   run:
     shell: bash
 
+env:
+  # Set to 'yes' to open a tunnel to GitHub's VMs through tmate on failures.
+  TMATE_DEBUG: 'no'
 
 jobs:
   testing:
-    runs-on: ubuntu-20.04
+    name: ${{ matrix.python-version }}-${{ matrix.job-name || 'lnx-default-tests' }}
```

Comment:
I think "linux" will be more clear in the UI than "lnx" and I don't think the extra 2 letters will make the names so long it becomes a problem.